packages/lending/src/adapter.test.ts

```diff
@@ -2,8 +2,33 @@ import { Adapter } from 'src/adapter';
 import * as common from '@protocolink/common';
 import { expect } from 'chai';
 import { filterPortfolio } from './protocol.utils';
+import { supportedChainIds } from './adapter.config';
+
+describe('Test Adapter', function () {
+  context('Test getProtocolInfos', function () {
+    supportedChainIds.forEach((chainId) => {
+      it(`network: ${common.toNetworkId(chainId)}`, async function () {
+        const adapter = await Adapter.createAdapter(chainId);
+
+        const protocolInfos = await adapter.getProtocolInfos();
+
+        protocolInfos.forEach((protocolInfo) => {
+          expect(protocolInfo).has.property('chainId', chainId);
+          expect(protocolInfo).has.property('protocolId');
+          expect(protocolInfo).has.property('marketId');
+
+          expect(Array.isArray(protocolInfo.reserveTokens)).to.equal(true);
```

Comment:
is it better to verify the array length > 0?

Reply:
fixed